generate-docs/scripts/postprocessor.ts

```diff
@@ -490,13 +490,19 @@ function cleanUpYmlFile(ymlFile: string, hostName: string): string {
                 remarks += ", ";
             }
         });
-        apiYaml.remarks += remarks;
+
```

Comment:
Can we update punctuation in line 486 to a period instead of a colon?  i.e., `Learn more about the types in this type alias through the following links.`